<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 54px;
background: #FFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Edit_token">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 13px;
border: 1px solid #1C1C1C;</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="find_string">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>335</y>
      <width>561</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 13px;
background: #F2F2F2;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Search_Button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>335</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 13px;
background: #1C1C1C;
color: #FFF;
font-family: Poppins;
font-size: 12px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>861</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="Music_search">
         <property name="font">
          <font>
           <family>Smooch Sans</family>
           <pointsize>-1</pointsize>
           <weight>87</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #1C1C1C;
leading-trim: both;
text-edge: cap;
font-family: Smooch Sans;
font-size: 60px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
         </property>
         <property name="text">
          <string>Music Search</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Error">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1C1C1C;
font-family: Poppins;
font-size: 20px;
font-style: normal;
font-weight: 400;
line-height: normal;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Data_Base">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 13px;
border: 1px solid #1C1C1C;</string>
    </property>
    <property name="text">
     <string>Db</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
